lib/cn/omf.h

```diff
@@ -258,7 +258,7 @@ struct cn_tstate_omf {
 
     uint32_t ts_khm_gen;
     uint32_t ts_khm_rsvd;
-    uint8_t  ts_khm_mapv[CN_TSTATE_KHM_SZ];
+    uint16_t ts_khm_mapv[CN_TSTATE_KHM_SZ];
```

Comment:
Should bump CN_TSTATE_VERSION from 1 to 2 and GLOBAL_OMF_VERSION from 2 to 3

Reply:
Yep.  Although it does matter much because the map size is checked before the version and bails.  But probably I should check the version first then the map size...   But this stuff is going away, so doesn't matter much, just trying to make it more accomodating to the interim changes...